rest_framework_filters/backends.py

```diff
@@ -27,8 +32,32 @@ def get_subset_class(view, queryset=None):
         yield
         self.get_filter_class = original
 
+    @contextmanager
+    def patch_for_rendering(self, request):
```

Comment:
This change is necessary to preserve form rendering in the API, now that the 'requested filters' override occurs during both form generation and queryset filtering. Previously, overriding the filters only occurred during queryset filtering, however this process would also generate the form if it wasn't already cached. If you cached the `form` before calling the queryset, it would not be generated with overridden filters.

This works by making the override context manager a noop.